bouncesaying.c

```diff
@@ -7,14 +7,12 @@
 
 #define FATAL "bouncesaying: fatal: "
 
-void main(argc,argv)
-int argc;
-char **argv;
+int main(int argc, char **argv)
 {
   int pid;
   int wstat;
 
-  if (!argv[1])
+  if (argc == 1 || !argv[1])
```

Comment:
I did not know there could be any situation `!argv[1] && argc ==1` could happen.

Reply:
I think just checking argc is good enough?

Reply:
It probably is. Let me do some experiments.